### DISCUSSION TITLE
Have you considered porting Hello DE to Adelie linux?

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#475] [Q&A]
Have you considered porting Hello DE to Adelie linux?
Because it's the only Linux I know of that can make this classic device work.

![adelie-imac](https://user-images.githubusercontent.com/44593430/215657852-98ceded1-1134-4ebf-be41-1736a3494856.jpg)
![IMG_2906](https://user-images.githubusercontent.com/44593430/215657861-c3af5a8a-5468-4ee4-bafa-c552d1314c29.jpg)

**Comment by probonopd** [comment]
helloSystem is based on FreeBSD, not Linux, and I am not interested in the work to maintain a Linux with helloDesktop at this time.

But maybe you can compile the helloDesktop components there.
